experiments/SlabLand/parameters_initialconditions.jl

```diff
@@ -0,0 +1,42 @@
+########
+# Set up parameters
+########
+parameters = (
+        xmax = 10000,   # horizontal extent of the SingleStack in the x direction [m]
+        ymax = 500,     # horizontal extent of the SingleStack in the y direction [m]
+        zmax = 10000,   # vertical extent of the SingleStack [m]
+        g    = 9.81,    # gravitation acceleration [m /s^2]
```

Comment:
worth using Clima parameters at this stage?

Reply:
I think for this illustrative example it's ok to leave the params list as is, but it's definitely worth doing for the more formal test cases that we will be publishing as part of CliMA. :)